docs/guides/creatingasource.md

```diff
@@ -91,15 +109,16 @@ metadata:
   name: sqs-guide
 spec:
   arn: arn:aws:sqs:us-east-1:123456789012:triggermesh
-  credentials:
-    accessKeyID:
-      valueFromSecret:
-        key: access_key_id
-        name: awscreds
-    secretAccessKey:
-      valueFromSecret:
-        key: secret_access_key
-        name: awscreds
+  auth:
+    credentials:
+      accessKeyID:
+        valueFromSecret:
+          key: access_key_id
+          name: awscreds
+      secretAccessKey:
+        valueFromSecret:
+          key: secret_access_key
+          name: awscreds
```

Comment:
nit picking: It might feel more logical while reading to see `name` before `key`, simply because a key is _part of_ the secret referenced by name, and not the other way around. Details.